lib/src/bridge/scanning_mixin.dart

```diff
@@ -1,19 +1,29 @@
 part of _internal;
 
 mixin ScanningMixin on FlutterBLE {
-  Stream<ScanResult>? _scanEvents;
-
-  Stream<ScanResult> _prepareScanEventsStream() {
-    return const EventChannel(ChannelName.scanningEvents)
-        .receiveBroadcastStream()
-        .handleError(
-          (errorJson) => throw BleError.fromJson(jsonDecode(errorJson.details)),
+  Stream<ScanResult>? __scanEvents;
+  Stream<ScanResult> get _scanEvents {
+    var scanEvents = __scanEvents;
+    if (scanEvents == null) {
+      scanEvents = 
+        const EventChannel(
+          ChannelName.scanningEvents
+        ).receiveBroadcastStream(
+        ).handleError(
```

Comment:
Has the formatter been run here? It looks weird.

Reply:
it's a style, it's different to your style, to me it's easier to overlook function chaining like this because `).foo()`easier to catch for the eye then simply `.foo()` wich as you start to read it can also be an enum case with a leading dot /not in dart ofc/.. I don't mind changing it to just leading dot style, but it already wasn't consistent anyways.

I do not like autoformatters (or i haven't found one that actually works well)

Reply:
pub.dev cuts points for code not formatted using the dartfmt. Like it or not, it's mandatory. 🙂